src/som/primitives/actors/PromisePrims.java

```diff
@@ -184,7 +216,7 @@ protected static final SPromise whenResolvedOrError(final SPromise rcvr,
       assert error.getMethod().getNumberOfArguments() == 2;
 
       Actor current = EventualMessage.getActorCurrentMessageIsExecutionOn();
-      SPromise  promise  = SPromise.createPromise(current);
+      SPromise  promise  = SPromise.createPromise(current, false, false, true);
```

Comment:
all the places where you pass predefined sets of `false` flags, are likely place where one might want to be able to set a breakpoint.
So, here, I think, we are at the place where the promise that is returned by a `whenResolved:`  is created.
So, actually, we probably would like to have promise breakpoints on `whenResolved:`, too.